### DISCUSSION TITLE
[Request] 恳求大佬给一个X.AI在VERCEL中部署的变量名称

### DISCUSSION THREAD

**Discussion by mingmenmama** [lobehub/lobehub#4774] [Self Hosting] [answered]
### 🥰 需求描述

恳求大佬给一个X.AI在VERCEL中部署的变量名称

### 🧐 解决方案

恳求大佬给一个X.AI在VERCEL中部署的变量名称

[a comment by an automated account is not shown]

**Comment by SpeedupMaster** [comment]
`XAI_API_KEY`

**Comment by mingmenmama** [comment]
> `XAI_API_KEY`

API是对的,但是MODEL是空的
尝试过XAI_MODEL_LIST
但是并不正确

[a comment by an automated account is not shown]

**Comment by SpeedupMaster** [comment]
> > `XAI_API_KEY`
> 
> API是对的,但是MODEL是空的 尝试过XAI_MODEL_LIST 但是并不正确

之前docker部署直接设置API KEY就可以了，vercel 没试过，照理来说应该是一样的

[a comment by an automated account is not shown]

**Comment by arvinxx** [comment]
`XAI_MODEL_LIST` 应该有效的才对， @hezhijie0327 帮忙看看？

[a comment by an automated account is not shown]

**Comment by hezhijie0327** [comment]
@arvinxx 这个 PR #4747 

环境变量之前这里 #4734 没声明，然后稍微改了下，后续就直接从runtime读了  

https://github.com/lobehub/lobe-chat/pull/4747/commits/5810203057f481f2ca9d2b554c2a220189fded52#diff-0fbc39fd0ef3d90d2c0db3d710364a69afce03cd86745f26fcae5716f962dc8a

**Comment by hezhijie0327** [comment]
哦 看错了... Model List

[a comment by an automated account is not shown]

**Comment by hezhijie0327** [comment]
vercel 没找到重启功能，在重新构建中，等个结果

PS: Docker 下正常
![image](https://github.com/user-attachments/assets/d694ad04-eaab-4ac7-b40d-8164a1869d6f)

**Comment by hezhijie0327** [comment, marked as answer]
@mingmenmama 变量更换后，重构下应该就好了，我试了下 `ACCESS_CODE` 逻辑也是一样的
![image](https://github.com/user-attachments/assets/ae91f222-48df-4e12-bc0c-6fbc9df53b49)
![image](https://github.com/user-attachments/assets/7bb1dbd1-cc14-4654-b8f4-30b706afd700)

[a comment by an automated account is not shown]
